homeassistant/components/androidtv_remote/manifest.json

```diff
@@ -7,6 +7,6 @@
   "integration_type": "device",
   "iot_class": "local_push",
   "loggers": ["androidtvremote2"],
-  "requirements": ["androidtvremote2==0.1.2"],
+  "requirements": ["androidtvremote2==0.2.0"],
```

Comment:
Please bump in a separate PR

Reply:
https://github.com/home-assistant/core/pull/139141